src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -240,6 +244,17 @@ if (__DEV__) {
               'or switching to a modern browser. If you suspect that this is ' +
               'actually an issue with React, please file an issue.',
           );
+        } else if (isCrossOriginError) {
+          error = new Error(
+            "A cross-origin error was thrown. React doesn't have access to " +
+              'the actual error because it catches errors using a global ' +
+              'error handler, in order to preserve the "Pause on exceptions" ' +
+              'behavior of the DevTools. This is only an issue in DEV-mode; ' +
+              'in production, React uses a normal try-catch statement.\n\n' +
+              "It's recommended to serve JavaScript files from the same " +
+              'origin as your application.',
```

Comment:
I think it's only if the error is thrown from a script other than the origin of the page...?

Look how @acdlite used the external `expect` library here. https://github.com/facebook/react/pull/10353/files#diff-f2b0545941b151d6a1f95ada1639d6f2R147